projects/aca-shared/store/src/actions/viewer.actions.ts

```diff
@@ -71,7 +71,7 @@ export class ClosePreviewAction implements Action {
 
 export class RefreshPreviewAction implements Action {
   readonly type = ViewerActionTypes.RefreshPreview;
-  constructor(public payload?: NodeEntry) {}
+  constructor(public node: unknown) {}
```

Comment:
as I said in this comment https://github.com/Alfresco/alfresco-content-app/pull/3669/files#r1505947023 - please set Node type here. 

It's generally good practice to have set type.

Reply:
yeah, I know, and I did it, but then there is an error which I sent above